lib/main_common.pm

```diff
@@ -3207,6 +3207,7 @@ sub load_kernel_baremetal_tests {
     }
     # make sure we always have the toolchain installed
     loadtest "toolchain/install";
+    loadtest "console/perf";
```

Comment:
Why schedule it also here/baremetal ?

Reply:
Because it is part of the progress issue:

- perf stat/record/list
- check/analyze perf record (perf.data)
- perf record command - record some basic commands
- **Do same on baremetal** - perf is hw/cpu dependant

Reply:
ehm ok, the test does not do anything that would have to run on bratemetal ... but whatever
Then schedule it after updates

Reply:
I realize my answer does not help much, sorry but that was all I knew as well. 
@plusky since you created the progress issue, could you please argument a little more for @dzedro on why scheduling the bare metal test as well? Thank you!

@dzedro To schedule it after the update do I have to move it right after `loadtest "kernel/build_git_kernel"` or pick a different place at all? Apologies for the confusion, I am just starting to learn all how to schedule tests in the right place!

Reply:
No problem, not sure how the install_ltp_baremetal behaves, looks like it does update only kernel.
I think to schedule perf update on http://openqa.qam.suse.cz you would have to update bot template https://gitlab.suse.de/qa-maintenance/metadata/-/blob/master/bot/bot.yml 
@mimi1vx will know better

Reply:
Hello @dzedro , Ondřej told me he has no idea either about a good place to schedule it.
Could you please suggest the exact place on where to put the test, or agree on leave it as it is for now? So far it works fine right after the toolchain install: http://10.161.229.249/tests/135#step/perf/41
Thank you!

Reply:
ok, leave it as it is